scripts/analyze-har-quests.py

```
@@ -0,0 +1,287 @@
+def query_keys(entry: dict[str, Any]) -> list[str]:
+    keys: list[str] = []
+    seen: set[str] = set()
+    for item in entry.get("request", {}).get("queryString", []) or []:
+        name = item.get("name")
+        if name and name not in seen:
+            seen.add(name)
+            keys.append(name)
+    return keys
+
+
+def parse_body(entry: dict[str, Any]) -> Any | None:
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `parse_body`

(ANN401)